deps.edn

```diff
@@ -1,6 +1,6 @@
 {:paths ["components/src" "src" "react-native/src/cljsjs" "resources"]
  :deps  {org.clojure/clojure         {:mvn/version "1.9.0"} ;; Keep in sync with .TOOLVERSIONS
-         org.clojure/clojurescript   {:mvn/version "1.10.439"}
+         org.clojure/clojurescript   {:mvn/version "1.10.238"}
```

Comment:
why downgrade?

Reply:
Somehow we used 2 different versions and there are some issues with using the more recent version.